index.yaml

```diff
@@ -4488,7 +5190,24 @@ entries:
       catalog.rancher.io/release-name: rancher-pushprox
     apiVersion: v1
     appVersion: 0.1.0
-    created: "2021-10-18T13:41:50.692613-07:00"
+    created: "2021-11-19T12:18:55.244654-07:00"
+    description: Sets up a deployment of the PushProx proxy and a DaemonSet of PushProx
+      clients.
+    digest: 66e43fdc64ef2311e404ab7db4c064e9aa743d5563bb3788b4fcccf7bcd59cae
+    name: rancher-pushprox
+    type: application
+    urls:
+    - assets/rancher-pushprox/rancher-pushprox-100.0.1.tgz
+    version: 100.0.1
```

Comment:
Those three new annotations are missing on this chart, but I think it is ok to leave it as it is for now since 
1) this chart is only used by monitoring as a sub-chart 
2) we still release it as a standalone chart for a special case where users might want to install it  (@aiyengar2 do you remember the exact use case? )

Reply:
I can update the chart to add those annotations if we want.

Reply:
rancher-pushprox is a hidden chart, so the only one it would need the `rancher-version` and even then, I don't believe we are doing that on sub-charts

Reply:
Good point, I forgot it is hidden.  In this case, no need for the annotations.